tokenLists/fantom.mainnet.tokenlist.json

```diff
@@ -576,16 +565,15 @@
             "_scan": "https://ftmscan.com/token/0x5a2e451Fb1b46FDE7718315661013ae1aE68e28C",
             "chainId": 250,
             "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/CGS.png"
+            "logoURI": "https://s2.coinmarketcap.com/static/img/coins/64x64/13106.png"
         },
         {
             "name": "Sing Token",
             "symbol": "SING",
             "address": "0x53D831e1db0947c74e8a52618f662209ec5dE0cE",
             "_scan": "https://ftmscan.com/token/0x53D831e1db0947c74e8a52618f662209ec5dE0cE",
             "chainId": 250,
-            "decimals": 18,
-            "logoURI": "https://assets.spookyswap.finance/tokens/SING.png"
+            "decimals": 18
```

Comment:
add logo

Reply:
The URL is no longer accessible and I couldn't find the logo URL in CoinMarketCap and ftmscan.com, so I removed it